src/Rules/AustrianInsuranceNumber.php

```diff
@@ -38,6 +38,7 @@ public function isValid(mixed $value): bool
         return is_numeric($value)
             && $this->startsNotWithZero($value)
             && $this->hasValidLength($value)
+            && $this->hasValidBirthday($value)
```

Comment:
I read in the [spec](https://www.sozialversicherung.at/cdscontent/?contentid=10007.820902&viewmode=content#datareplacementdoneWorausbestehtdieVersicherungsnummer820902) that the date of birth does not always appear in the number. Does this mean that these particular numbers will no longer be recognized as valid after the patch?

Reply:
The date of birth must always be stated. If it is not known exactly, the day is always the 1st, the month may be greater than 12.
If the date of birth is missing, the number is invalid.